nestkernel/event.cpp

```diff
@@ -50,6 +52,26 @@ Event::Event()
 {
 }
 
+index
+Event::retrieve_sender_node_id_from_source_table() const
+{
+  if ( sender_node_id_ > 0 )
+  {
+    return sender_node_id_;
+  }
+  else
+  {
+    const index node_id = kernel().connection_manager.get_source_node_id(
+      sender_spike_data_.get_tid(), sender_spike_data_.get_syn_id(), sender_spike_data_.get_lcid() );
+    return node_id;
+  }
+}
+
+index
+Event::get_receiver_node_id( void ) const
```

Comment:
`void` argument is redundant here.
```suggestion
Event::get_receiver_node_id() const
```